socorro/unittest/processor/test_processor_app.py

```diff
@@ -277,6 +277,48 @@ def mocked_save_raw_and_processed(*_):
         assert captured_exception.__class__ == ValueError
         assert captured_exception.args[0] == 'Someone is wrong on the Internet'
 
+    @mock.patch('socorro.lib.raven_client.raven')
+    def test_transform_get_error_with_raven_configured_successful(self, mock_raven):
+        # First mock everything so we're not sending real things anywhere
+        captured_exceptions = []  # a global
+
+        def mock_capture_exception(exc_info=None):
+            captured_exceptions.append(exc_info)
+            return 'someidentifier'
+
+        raven_mock_client = mock.MagicMock()
+        raven_mock_client.captureException.side_effect = mock_capture_exception
+
+        mock_raven.Client.return_value = raven_mock_client
+
+        # Set up a processor and mock .get_raw_crash() in that
+        config = self.get_standard_config(
+            sentry_dsn='https://abc123@example.com/project'
+        )
+        pa = ProcessorApp(config)
+        pa._setup_source_and_destination()
+
+        def mocked_get_raw_crash(*_):
+            raise ValueError('simulated error')
+
+        pa.source.get_raw_crash.side_effect = (
+            mocked_get_raw_crash
+        )
+
+        # Note that the processor silently ignores .get_raw_crash() and
+        # .get_raw_dumps_as_files() errors and only logs stuff with the
+        # assumption that they're ephemeral
```

Comment:
Did you mean to say that it ignores errors when the crash ID isn't found here? Otherwise isn't this false?

Reply:
The code that calls `get_raw_crash` and `get_raw_dumps_as_files` catches exceptions from them and rejects the raw crash if they're not an error about the crash ID not being found, but this comment implies that it should instead silently ignore those errors. That seems to be a contradiction.

Reply:
I don't see how this is a contradiction and I'm not following what you're saying. I feel stupid and don't want to belabor this. I don't want to have obtuse comments--that doesn't help anyone. I'll try rewording it and maybe that'll be better.